Sources/AirbnbSwiftFormatTool/AirbnbSwiftFormatTool.swift

```diff
@@ -41,11 +44,25 @@ struct AirbnbSwiftFormatTool: ParsableCommand {
     try swiftLint.run()
     swiftLint.waitUntilExit()
 
-    guard
-      swiftFormat.terminationStatus == 0,
-      swiftLint.terminationStatus == 0
-    else {
-      throw LintError.lintFailure
+    if log {
+      log(swiftFormat.shellCommand)
+      log(swiftLint.shellCommand)
+      log("SwiftFormat ended with exit code \(swiftFormat.terminationStatus)")
+      log("SwiftLint ended with exit code \(swiftLint.terminationStatus)")
+    }
+
+    // SwiftFormat ends with exit code 1 on lint failure, and SwiftLint ends with exit code 2
+    if swiftFormat.terminationStatus == 1 || swiftLint.terminationStatus == 2 {
```

Comment:
nit: we could make some enums for tool error statuses to make this self-documenting
```swift
    if swiftFormat.terminationStatus == SwiftFormatTerminationStatus.lintFailue {
```

Reply:
Good idea!